simple_salesforce/tests/test_api.py

```diff
@@ -29,10 +30,433 @@
     SalesforceExpiredSession,
     SalesforceRefusedRequest,
     SalesforceResourceNotFound,
-    SalesforceGeneralError
+    SalesforceGeneralError,
+    SFType
 )
 
 
+def _create_sf_type():
+    """Creates SFType instances"""
+    return SFType(
+        object_name='Case',
```

Comment:
Might it be worthwhile to have these attributes configurable through input parameters?